src/http/tls.rs

```diff
@@ -36,29 +33,24 @@ pub fn init_tls_client(tls: TlsData) {
     *CONNECTOR.lock().unwrap() = Some(Arc::new(TlsConnector::new(tls)));
 }
 
-
-fn load_p12(p12_path: &str, p12_pass: &str) -> Result<ParsedPkcs12, ErrorStack> {
-    let mut file = File::open(p12_path)
-        .unwrap_or_else(|err| panic!("couldn't open p12 file: {}", err));
+fn load_p12(p12_path: &str, p12_pass: &str) -> ParsedPkcs12 {
+    let mut file = File::open(p12_path).unwrap_or_else(|err| panic!("couldn't open p12 file: {}", err));
```

Comment:
`unwrap_or_else(|err| panic!("couldn't open p12 file: {}", err))` can be replaced by `expect("couldn't open p12 file")`, no?